images/android-ndk-r22-jdk17

```diff
@@ -50,8 +50,10 @@ RUN set -eu \
         "platforms;android-28" \
         "extras;android;m2repository" \
         "extras;google;m2repository" \
-        "extras;m2repository;com;android;support;constraint;constraint-layout;1.0.2" \
```

Comment:
any reason why this line is removed?

Reply:
It's discontinued, and `sdkmanager` cannot find it anymore.